Change NCBI Parameter to Nextflow Secret (#124)
nf-core/metapep: docs/usage.md

### Input type taxa

> The input type taxa allows the user to specify a taxid on strain level, for which all proteins are downloaded. To be able to download data from Entrez in an automatic pipeline a registered account needs to be specified, and the login email alongside an API key needs to be available to nextflow to use the download taxa path of nf-core/metapep. To set the environment secrets available to nextflow follow these steps:

Reviewer: ```suggestion
The input type taxa allows the user to specify a taxid on strain level, for which all proteins are downloaded. To enable the pipeline to download data from Entrez a registered NCBI account is needed, and the login email alongside an API key need to be provided. To set the environment secrets available to Nextflow follow these steps:
```
